### ISSUE TITLE
need tf.random.shuffle supported.

### ISSUE THREAD

**Issue by chan4cc** [onnx/onnx#3451]
# New Operator

### Describe the operator
Why is this operator necessary? What does it accomplish?

**This is a frequently used operator in `tensorflow/keras`**

### Can this operator be constructed using existing onnx operators?
If so, why not add it as a function?

 I don't know.

### Are you willing to contribute it? (Y/N)
N. I am still a novice to onnx and C++.

### Notes
Any additional information

 I'm trying to convert a keras h5 model(create with tf1.12) to ONNX:
```python
from tensorflow.keras.models import load_model

model = load_model(my_model_file)

onnx_model = keras2onnx.convert_keras(model, model.name)
onnx_model.save_model('model.onnx')

```
which report a error like:
```
WARN: No corresponding ONNX op matches the tf.op node cos_sim/RandomShuffle_6 of type RandomShuffle
      The generated ONNX model needs run with the custom op supports.
WARN: No corresponding ONNX op matches the tf.op node cos_sim/RandomShuffle_5 of type RandomShuffle
      The generated ONNX model needs run with the custom op supports.
WARN: No corresponding ONNX op matches the tf.op node cos_sim/RandomShuffle_4 of type RandomShuffle
      The generated ONNX model needs run with the custom op supports.
WARN: No corresponding ONNX op matches the tf.op node cos_sim/RandomShuffle_3 of type RandomShuffle
      The generated ONNX model needs run with the custom op supports.
WARN: No corresponding ONNX op matches the tf.op node cos_sim/RandomShuffle_2 of type RandomShuffle
      The generated ONNX model needs run with the custom op supports.
WARN: No corresponding ONNX op matches the tf.op node cos_sim/RandomShuffle_1 of type RandomShuffle
      The generated ONNX model needs run with the custom op supports.
WARN: No corresponding ONNX op matches the tf.op node cos_sim/RandomShuffle of type RandomShuffle
      The generated ONNX model needs run with the custom op supports.
There is an error(<class 'StopIteration'>) happened during optimizing on the converted model!
```
Thers is a `tf.random.shuffle` op used in my code. And I tried pass param `custom_op_conversions` with `{'RandomShuffle': tf.random.shuffle}`, and it report the same error. it seems that `onnx` opset not support `op tf.random.shuffle` yet?

**Version Used:**
tensorflow: 1.12
onnxconverter-common: 1.8.1
keras2onnx: 1.8.0
onnx: 1.8.1

I've also searched `RandomShuffle` in file `onnx/onnx/defs/operator_sets.h` and indeed did not find its existence up to version 14.

**Comment by jcwchen** [comment]
Hi @chan4cc,
Yes shuffle is not supported yet... I will tag it as an enhancement.

**Comment by milind-soni** [comment]
> Hi @chan4cc,
> Yes shuffle is not supported yet... I will tag it as an enhancement.

How do we go about implementing this.

**Comment by jcwchen** [comment]
Hi @milind-soni,
Actually I think tensorflow-onnx just supported this conversion recently: https://github.com/onnx/tensorflow-onnx/pull/1658 and it was included in tensorflow-onnx 1.9.2: https://github.com/onnx/tensorflow-onnx/releases/tag/v1.9.2. Perhaps you can give it a try and we can close this issue if it works. Thank you!

**Comment by jcwchen** [comment]
I think it should be supported in tf2onnx now. Close here.
